src/theme.ts

```diff
@@ -1,8 +1,8 @@
 import { createElement } from 'react';
-import { createMuiTheme } from '@material-ui/core/styles';
+import { createTheme } from '@material-ui/core/styles';
 
 // Create a theme instance.
-const theme = createMuiTheme({
+const theme = createTheme({
     overrides: {
```

Comment:
Hey sorry that I didn't get a chance to look at this before! Why did you change this? I have to change it back in my current branch in order for it to compile